cmd/wallet-web/src/components/CredentialCards/GeneralCard.vue

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+<template>
```

Comment:
is this for credentials which doesn't fall under category?

Reply:
yes